core/src/test/scala/unit/kafka/cluster/PartitionTest.scala

```diff
@@ -36,7 +35,7 @@ import org.apache.kafka.metadata.LeaderRecoveryState
 import org.junit.jupiter.api.Assertions._
 import org.junit.jupiter.api.Test
 import org.mockito.ArgumentMatchers
-import org.mockito.ArgumentMatchers.{any, anyBoolean, anyInt, anyLong, anyString}
+import org.mockito.ArgumentMatchers.{any, anyBoolean, anyInt, anyLong}
```

Comment:
Can you also remove the ZooKeeper bits in `testIsrNotExpandedIfReplicaIsFencedOrShutdown()`?